### PR TITLE
Pull request for Docker image

### PR DESCRIPTION
**Pull request by yuukiiwa**
Hi @drpatelh, this pull request is for updating the docker image with new packages added to environment.yml.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -13,6 +13,9 @@ dependencies:
   - conda-forge::pigz=2.3.4
   - conda-forge::psutil=5.7.0
```
Comment (drpatelh) [inline environment.yml:15]:
```suggestion
```

**Review by drpatelh — CHANGES_REQUESTED** [review]
Thanks @yuukiiwa 🙂 Can we add these packages to the `CHANGELOG` too please. See [here](https://github.com/nf-core/atacseq/blob/master/CHANGELOG.md#dependencies).

**Comment by yuukiiwa** [comment]
Hi @drpatelh, thanks for the suggestion!  I have added the packages to the `CHANGELOG`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,14 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
-* Update template to tools `1.10.2`
```
Comment (drpatelh) [inline CHANGELOG.md:10]: Why did you remove this line @yuukiiwa ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,14 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+* Add r-devtools `2.3.0`
+* Add stringtie `2.0`
+* Add subread `2.0.1`
+* Add bioconductor-deseq2 `1.28.0`
+* Add bioconductor-drimseq `1.16.0`
+* Add bioconductor-dexseq `1.34.0`
+* Add bioconductor-stager `1.10.0`
+* Add bioconductor-bsgenome `1.56.0`
+
 ### `Deprecated`
```
Comment (drpatelh) [inline CHANGELOG.md:26]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,14 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
```
Comment (drpatelh) [inline CHANGELOG.md:27]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,15 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
```
Comment (drpatelh) [inline CHANGELOG.md:9]:
```suggestion

* Update template to tools `1.10.2`
```

**Review by drpatelh — APPROVED** [review]
Thanks @yuukiiwa :) I made the final few changes on the PR so it can be merged 👍

File: CHANGELOG.md (hunk 2 above)
Comment (yuukiiwa) [reply CHANGELOG.md:10]: I am sorry, @drpatelh! I did that accidentally...

Comment (drpatelh) [reply CHANGELOG.md:10]: No worries :) `dev` container has been built and pushed to Dockerhub now so you can re-pull it and it should work.

Comment (yuukiiwa) [reply CHANGELOG.md:10]: Thanks! Will do that soon!
